.github/workflows/versionControlAndAuditCheck.yml

```diff
@@ -393,10 +393,27 @@ jobs:
           echo "This step will make sure that an audit is logged for each contract modified/added by this PR."
           echo "It will also make sure that no information is missing in the audit log and that the information is meaningful."
 
-          # load list of protected contracts
-          PROTECTED_CONTRACTS=$(cat contracts_for_audit.txt)
+          echo "cat contracts_for_audit.txt:"
+          echo cat contracts_for_audit.txt
+          echo ""
+
+          # convert from comma-separated to newline-separated
+          CONTRACTS_FOR_AUDIT_STR=$(tr ',' '\n' < contracts_for_audit.txt)
+          echo "CONTRACTS_FOR_AUDIT_STR: $CONTRACTS_FOR_AUDIT_STR"
+          PROTECTED_CONTRACTS="$CONTRACTS_FOR_AUDIT_STR"
           echo "PROTECTED_CONTRACTS: $PROTECTED_CONTRACTS"
 
+
+
+          # CONTRACTS_FOR_AUDIT_STR=$(IFS=$'\n'; $(cat contracts_for_audit.txt))
+          # echo "CONTRACTS_FOR_AUDIT_STR: $CONTRACTS_FOR_AUDIT_STR"
+          # echo -e "$CONTRACTS_FOR_AUDIT_STR" > contracts_for_audit.txt
+          # echo ""
+
+          # # load list of protected contracts
+          # PROTECTED_CONTRACTS=$(cat contracts_for_audit.txt)
+          # echo "PROTECTED_CONTRACTS: $PROTECTED_CONTRACTS"
+
```

Comment:
can we remove commented code?

Reply:
I would like to keep it for now until the flow is fully implemented